src/frequenz/sdk/timeseries/ev_charger_pool/_set_current_bounds.py

```diff
@@ -95,17 +95,16 @@ async def _run(self) -> None:
         ).into_peekable()
         latest_bound: Dict[int, ComponentCurrentLimit] = {}
 
-        select = Select(
-            bound_chan=self._bounds_rx,
-            timer=Timer(self._repeat_interval.total_seconds()),
-        )
-        while await select.ready():
+        bound_chan = self._bounds_rx
+        timer = Timer.periodic(timedelta(self._repeat_interval.total_seconds()))
```

Comment:
I think this should be `Timer.timeout()` or use `skip_missed_ticks=True` for `periodic()`, because I guess we don't want to send the bounds again as burst if the loop is delayed for too long, right @shsms ?

Reply:
oh true,  I guess `timeout` definitely makes more sense here.  Same for the two timers created in the `__init__` method of the `BatteryStatusTracker` class.